module/applications/compendium-browser.mjs

```diff
@@ -481,11 +481,16 @@ export default class CompendiumBrowser extends Application5e {
     const { max, min } = this.options.selection;
 
     context.displaySelection = this.displaySelection;
-    context.invalid = (value < (min || -Infinity)) || (value > (max || Infinity)) ? "invalid" : "";
+    context.invalid = (value < (min || -Infinity)) || (value > (max || Infinity));
     const suffix = this.#selectionLocalizationSuffix;
     context.summary = suffix ? game.i18n.format(
       `DND5E.CompendiumBrowser.Selection.Summary.${suffix}`, { max, min, value }
     ) : value;
+    const pr = new Intl.PluralRules(game.i18n.lang);
```

Comment:
We're trying to reply on the utility `getPluralRules` from `utils.mjs` these days (didn't exist when `#onHandleSubmit` was written), so you can import that and swap this out:

```suggestion
// Above: import { getPluralRules } from "../utils.mjs";
    const pr = getPluralRules();
```